src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -957,7 +969,7 @@ internal static Attribute[] ReflectGetAttributes(Type type)
                 {
                     // Get the type's attributes.
                     //
-                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).ToArray();
+                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).OfType<Attribute>().ToArray();
```

Comment:
What's the purpose of this change?  GetCustomAttributes returns an Attribute[], so isn't this OfType effectively a nop?  Or could the array contain nulls or something you're trying to weed out?